### ISSUE TITLE
`Using the Fetch API` doc has partially incorrect comment about default `credentials`

### ISSUE THREAD

**Issue by ppena-LiveData** [mdn/content#14820]
### MDN URL

/en-US/docs/Web/API/Fetch_API/Using_Fetch

### What information was incorrect, unhelpful, or incomplete?

There is a comment that says, "The spec changed the default credentials policy to `same-origin`." It turns out that that sentence is partially correct but partially incorrect. Via experimentation, I found that the browser uses the `credentials` option to `fetch()` not just to know whether to send credentials/cookies, but it also uses it to know whether to set the cookie that the web server responds with. If `credentials: 'same-origin'` is explicitly passed to `fetch()`, then the `Set-Cookie` will be correctly handled, but if it is not explicitly passed to `fetch()`, the `Set-Cookie` will just be ignored.

### What did you expect to see?

I am not sure what section this info should be added, but it is very confusing when the `Set-Cookie` is just ignored, so this documentation should definitely mention that caveat somewhere.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/api/fetch_api/using_fetch`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/fetch_api/using_fetch/index.md
* Last commit: https://github.com/mdn/content/commit/c4b6112d856dbb5fa7b929e5034499c605a51e08
* Document last modified: 2022-03-28T14:52:22.000Z

</details>

**Comment by sideshowbarker** [comment]
> If `credentials: 'same-origin'` is explicitly passed to `fetch()`, then the `Set-Cookie` will be correctly handled, but if it is not explicitly passed to `fetch()`, the `Set-Cookie` will just be ignored.

Do you have a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) which demonstrates that?

Per the requirements in the Fetch spec, for same-origin requests, the value of the `credentials` option is already set to `same-origin`, and you should not need to explicitly set it. 

So, if you’re making a same-origin `fetch()` call, and the browser is ignoring any `Set-Cookie` response header, then that’s either a browser bug, or else I guess there’s something else happening that’s causing the `credentials` option to be set to `omit` in the `fetch()` call.

> the browser uses the `credentials` option to `fetch()` not just to know whether to send credentials/cookies, but it also uses it to know whether to set the cookie that the web server responds with

Yes — https://developer.mozilla.org/en-US/docs/Web/API/fetch#credentials documents that the `same-origin` value for the `credentials` option _“Tells browsers to include credentials with requests to same-origin URLs, and use any credentials sent back in responses from same-origin URLs”_.

**Comment by ppena-LiveData** [comment]
I was trying all sorts of things, and I must've mixed up and thought I had used `credentials: 'same-origin'` when I actually had used `credentials: 'include'` with the separate subdomain API endpoint I was hitting, since I've tried again, and `same-origin` does not work with the APIs on the separate subdomain. I think I also confused myself since the cookie domain value also includes subdomains. Whatever it was, this is wrong, so I'm closing this issue. Thanks.
